Storage/test.cc

```diff
@@ -308,6 +420,21 @@ TEST(TransactionalStorage, SmokeTest) {
       EXPECT_EQ(2, Value(fields.m.Get(2, "two")).phew);
       EXPECT_EQ(3, Value(fields.m.Get(2, "too")).phew);
       EXPECT_FALSE(Exists(fields.m.Get(3, "three")));
+
+      EXPECT_FALSE(fields.o.Empty());
+      EXPECT_EQ(2u, fields.o.Size());
+      EXPECT_EQ(1, Value(fields.o.Get(1, "one")).phew);
+      EXPECT_EQ(1, Value(fields.o.GetCol("one")).phew);
+      EXPECT_EQ(4, Value(fields.o.Get(3, "too")).phew);
+      EXPECT_EQ(4, Value(fields.o.GetRow(3)).phew);
+      EXPECT_FALSE(Exists(fields.o.Get(2, "two")));
+      EXPECT_FALSE(Exists(fields.o.GetRow(2)));
+      EXPECT_FALSE(Exists(fields.o.GetCol("two")));
+      EXPECT_FALSE(Exists(fields.o.Get(2, "too")));
+      EXPECT_TRUE(Exists(fields.o.GetCol("too")));
+      EXPECT_FALSE(Exists(fields.o.GetCol("three")));
+      EXPECT_FALSE(fields.o.Cols().Has("three"));
+      EXPECT_TRUE(fields.o.Cols().Has("too"));
     }).Wait();
```

Comment:
@mzhurovich @dkorolev Should I change `.Wait()` to `.Go()` and use the same style as above? Like:
`const auto result = ...`
`EXPECT_FALSE(WasCommitted(result));`